frontend/lib/identification/identification_page.dart

```diff
@@ -45,14 +46,21 @@ class IdentificationPageState extends State<IdentificationPage> {
           return Container();
         }
 
+        final applicationUrl = getApplicationUrl(context);
         if (userCodeModel.userCodes.isNotEmpty) {
           final List<Widget> carouselCards = [];
           for (var code in userCodeModel.userCodes) {
+            final applicationUrlForCardExtension = isCardExtendable(code.info, code.cardVerification)
```

Comment:
```suggestion
            final applicationUrl = isCardExtendable(code.info, code.cardVerification)
```
:upside_down_face: I think it would make more sense. because it will be 'for extension' only is the card is extendable. but the same variable is also used when the card is not extendable.